src/SearchResults.js

```diff
@@ -14,14 +15,29 @@ export default function Search({
   fetchSearchResults,
   showHeader = false,
 }) {
+  // eslint-disable-next-line camelcase
+  const page_size = 5;
+  const [
+    searchOptionsForSearchResults,
```

Comment:
naming suggestion: `searchOpts`. [Principle number 4: Use shorthands when possible](https://www.notion.so/Naming-Things-1b415d3a648b4a3dbb50bc7c1e032d75#e84e31a302f2402982b6235edfe8930e). Also goes against [principle 1](https://www.notion.so/Naming-Things-1b415d3a648b4a3dbb50bc7c1e032d75#fb2e34e44019446dbec8e3e74ab9d641), names with duplicate context should be shortened. `searchOptionsForSearchResults` contains name `SearchResults`, which is already implied by the filename, which provides the same context.

Reply:
3a615ad addresses this. I'm going to resolve this conversation because I don't think there are any further changes to make.